config_defaults_inc.php

```diff
@@ -1154,15 +1154,19 @@
 $g_complete_date_format = 'Y-m-d H:i T';
 
 /**
- * jscalendar date format string
- * go to http://www.php.net/manual/en/function.date.php
- * for detailed instructions on date formatting
+ * jquery-ui datepicker date format string
+ * @global string $g_calendar_js_date_format
+ */
+$g_calendar_js_date_format = 'yy-mm-dd';
+
+/**
+ * jquery-ui datepicker time addon time format string
  * @global string $g_calendar_js_date_format
```

Comment:
This variable name doesn't match the real variable.  Also if you rename the other one, then you need to update the similar phpdoc tag as well.